cluster/libvirt-coreos/util.sh

```diff
@@ -140,6 +143,18 @@ function initialize-pool {
       render-template "$ROOT/skydns-rc.yaml"  > "$POOL_PATH/kubernetes/addons/skydns-rc.yaml"
   fi
 
+  mkdir -p "$POOL_PATH/etcd"
+  if [[ ! -f "$ROOT/etcd-v2.0.9-linux-amd64.tar.gz" ]]; then
+      wget -P "$ROOT" https://github.com/coreos/etcd/releases/download/v2.0.9/etcd-v2.0.9-linux-amd64.tar.gz
+  fi
+  if [[ "$ROOT/etcd-v2.0.9-linux-amd64.tar.gz" -nt "$POOL_PATH/etcd/etcd" ]]; then
```

Comment:
Why hardcode which version of etcd is using here? Also we already run etcd server as docker image, why not pulling the image directly?